project/app/static/js/editGroupModal.js

```diff
@@ -1,143 +1,108 @@
-var selectedStudent = {};
-var selectedElements = [];
-
 //remove student/students from group
 function moveL() {
   //group students in form input
   var students = document.getElementById('id_students');
-  var list = students.value.split(',');
+  //group's student list
+  var studentList = document.getElementById('studentList');
```

Comment:
Tän olisin voinu sanoo jo aiemmin mut tuli vasta nyt mieleen, että nykyään JavaScriptissä on hyvien käytäntöjen mukaista käyttää "let" ja "const" identifiereita "var":in sijaan. Eli "let" jos kyseessä on muuttuja jota halutaan muokata myöhemmin, ja "const" jos kyseessä on vakio.